README.md

```diff
@@ -25,23 +25,44 @@ This can be useful in several ways:
 * Use within CI to perform integration testing.
 * Use as a local development environment for IDP engineers.
 
-## Quickstart:
+## Quickstart
 
-### Running the idpbuilder
+### Download and install the idpbuilder
 
-To get started, run this command:  
+Download the latest release with the commands:
 
+```bash
+version=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/cnoe-io/idpbuilder/releases/latest)
+version=${version##*/}
+curl -L -o ./idpbuilder.tar.gz "https://github.com/cnoe-io/idpbuilder/releases/download/${version}/idpbuilder-$(uname | awk '{print tolower($0)}')-$(uname -m | sed 's/x86_64/amd64/').tar.gz"
+tar xzf idpbuilder.tar.gz
+
+./idpbuilder  version
```

Comment:
2 spaces between idpbuilder and version

Reply:
sorry, saw this a tad bit late. will push a quick fix for this.